frontend/core-ui/src/modules/logs/components/filters/LogRecordTableFilterBars.tsx

```
@@ -60,80 +60,69 @@
+          </Popover.Trigger>
+          <Popover.Content>
+            <LogActionsFilter />
+          </Popover.Content>
+        </Popover>
+      </Filter.BarItem>
+
+      <SelectMember.FilterBar queryKey="userIds" />
 
       {customFilters.map(({ name, value }) => (
-        <Filter.BarItem>
+        <Filter.BarItem queryKey={name}>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)